saw-core/prelude/Prelude.sawcore

```diff
@@ -903,6 +903,7 @@ primitive expByNat : (a:sort 0) -> a -> (a -> a -> a) -> a -> Nat -> a;
 
 primitive equalString : String -> String -> Bool;
 
+primitive appendString : String -> String -> String;
```

Comment:
@Ptival: I believe `saw-core-coq` has some special treatment for the `appendString` function, so I'm not sure if adding it as a primitive here will cause problems.